<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="5000" />
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="moveo_nozzle_link" />
  <arg name="timeout" default="5" />
  <arg name="eps" default="1e-5"/>
  <arg name="num_threads" default="16"/>
  <arg name="model" default="$(find moveo_urdf)/urdf/moveo_urdf.urdf"/>
  <param name="robot_description" textfile="$(arg model)" />
  
  <node name="moveo_moveit_move_group" pkg="moveo_moveit_move_group" type="parallel_ik_test" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="eps" value="$(arg eps)" type = "double"/>
    <param name="num_threads" value="$(arg num_threads)" type = "int"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
